LevelUp.Pos.ProposedOrders.Tests/CalculatorTests.cs

```diff
@@ -60,25 +60,5 @@ private void RunTestArray(int[,] values)
                     i, expectedCheckValues, actualCheckValues, Environment.NewLine);
             }
         }
-
-        [TestMethod]
-        public void TestTest123()
```

Comment:
Not really, it is just one case when TestBattery stuff goes through multiple cases.